méli-mélo/2024-04-stepsquiz/meta.md

```diff
@@ -0,0 +1,40 @@
+---
+feature: méli-mélo
+"@context":
+  "@version": 1.0
+  dct: http://purl.org/dc/terms/
+  title:
+    "@id": dct:title
+    "@container": "@language"
+  description:
+    "@id": dct:description
+    "@container": "@language"
+  modified: dct:modified
+
+title:
+  en: Quiz pattern
+description:
+  en: Apply a quiz pattern to the existing Steps Form component.
+modified: 2024-04-16
+componentName: 2024-04-stepsquiz
+sponsor: Principal Publisher - ESDC
+
+pages:
+  examples:
+    - title: Quiz
+      language: en
+      path: index.html
+
+implementationPlan:
```

Comment:
That implementation plan is incomplete. It must contain all the key/main milestones required to have this experimenation transformed into a stable functionality. For example, the stabilization work of the provisional steps form in wet-boew is missing in that plan. It should also include translation, accessibility testing, usability testing....

Reply:
Information has been added please review and let me know.

Reply:
Look good. Thanks